modules/oadp-backing-up-applications-restic.adoc

```diff
@@ -10,6 +10,11 @@ You back up Kubernetes resources, internal images, and persistent volumes with R
 
 You do not need to specify a snapshot location in the `DataProtectionApplication` CR.
 
+[IMPORTANT]
+====
+Restic does not support backing up `hostPath` volumes. For more information, see link:https://{velero-domain}/docs/v{velero-version}/restic/#limitations[additional Rustic limitations].
```

Comment:
I believe this is meant to be fixed after the other PR gets merged, so ignore if that the case. But I wanted to bring up just to make sure, the link brings me to a "Page not found" 

https://velero.io/docs/v%7Bvelero-version%7D/restic/#limitations

Reply:
@kelbrown20 -- yes, that's why I requested in the description that the other PR be merged first. I will change the description to make this clearer.